playbooks/openshift-checks/pre-install.yml

```diff
@@ -0,0 +1,15 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- hosts: OSEv3
+  # Temporarily reverting to OSEv3 until group standardization is complete
```

Comment:
Same here, this comment is not useful unless someone is reading the Git history, and if so, the commit logs and PRs where the changes happened tell a better story.